CHANGELOG.md

```diff
@@ -2,6 +2,99 @@
 
 ## 5.14.8
 
+<!-- generated comparing v5.14.8..master -->
+
+_Sep 11, 2023_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Added `Drawer` component to Joy UI (#38169) @mnajdova
+- ✨ Added `firstButton`, `lastButton`, and `middleButton` classes to `ButtonGroup`'s button descendants in Material UI (#38520) @ZeeshanTamboli
+
+### `@mui/material@5.14.9`
+
+- &#8203;<!-- 44 -->[ButtonGroup][material-ui] Determine first, last and middle buttons to support different elements with correct styling (#38520) @ZeeshanTamboli
```

Comment:
Maybe we shouldn't remove the product name when it's already under the header.

```suggestion
- &#8203;<!-- 44 -->[ButtonGroup] Determine first, last and middle buttons to support different elements with correct styling (#38520) @ZeeshanTamboli
```

x the other pages.

Reply:
@mnajdova what do you think?

Reply:
I think it should be removed.

Reply:
Yep, let's remove it.

Reply:
I removed the product names when the change is already under the header 👍🏼 
Should we do the same for the Docs section (remove `[docs]`) and Core section (remove `[core]`)? 🤔

Reply:
Let's keep it as is now, we can decide this elsewhere so that we don't block this PR. As far as I could see the original point was around Material UI, which is already resolved.